.github/workflows/test.yml

```diff
@@ -44,18 +44,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [16, 18, 20]
-        npm: [8, 9, 10]
-        exclude: # node 16 is not compatible with npm 10
-          - node: 16
-            npm: 10
+        node: [20, 22]
+        npm: [10, 10]
```

Comment:
Node 18 is still in LTS for 5 more months. What is the timeline for this upgrade?

Reply:
Good call. I believe we have to wait til MRT officially supports Node 22. I think at least one more RP to start this upgrade.